packages/graph/hash_graph/lib/graph/src/store/postgres/query/statement/select.rs

```diff
@@ -12,6 +13,7 @@ pub struct SelectStatement<'q> {
     pub from: Table,
     pub joins: Vec<JoinExpression<'q>>,
     pub where_expression: WhereExpression<'q>,
+    pub order_by_expression: OrderByExpression<'q>,
```

Comment:
I assume this only works for versions at the moment, what would happen if I tried putting the expression somewhere else in the query, would it break, work, throw an error?

Reply:
Oh we're not exposing this to users through the actual query objects I take it? It's implicit in the `latest` special-case?

Reply:
No, we are not. It's also not special-cased in any way. There was a special case when using `SELECT DISTINCT` (next PR), but that also got removed.